### DISCUSSION TITLE
pdf++ for ipad

### DISCUSSION THREAD

**Discussion by Kenotion** [RyotaUshio/obsidian-pdf-plus#284] [Q&A]
今天在ipad上的ob中下载了PDF++，比之前好多了不再提示不兼容可以下载了，就是下载后不能运行！！？？

**Comment by RyotaUshio** [comment]
Can you repost it in English (or Japanese)? I tried auto-translate but I didn't understand.

**Reply by Kenotion** [reply]
PDF++ has been downloaded in Obsidian on iPad. It can be downloaded, but it cannot run after downloading!!??

**Reply by RyotaUshio** [reply]
How about updating your iPadOS?

**Reply by Kenotion** [reply]
iPadOS has been upgraded to iPadOS 18.2 beta 4.

**Reply by Kenotion** [reply]
The new version of PDF++ is already available for use on the iPad and it works great! 👍 However, I've found a small issue. The mouse wheel can't be used to scroll up and down to turn pages in PDF documents.

**Reply by RyotaUshio** [reply]
Glad to hear the issue is fixed!

The scrolling issue is in fact not PDF++'s bug, but it's Obsidian's.

https://forum.obsidian.md/t/ipados-ios-unable-to-scroll-pdfs-with-external-mouse-or-touchpad/87596

**Reply by Kenotion** [reply]
Yes. It seems that we can only wait for the fixing progress in the next version of Obsidian.

**Reply by Kenotion** [reply]
Yes, for the ob1.8.9 version, in PDF++, the mouse can already be used to turn the pages of PDF documents on an iPad. I hope that the Chinese-language interface of the PDF++ settings can be launched as soon as possible. The plugin is already doing a really great job. 👍

**Comment by padanellop** [comment]
Hi, I have the same issue. It works well sync between windows pc and iPhone; but when I try to use it on my IPad, it let me install but when I try to enable, it say: failed to load PDF++. Thank you :) 
![013C3171-CECA-4E03-B752-2C5FFE19BD0D](https://github.com/user-attachments/assets/33f26753-f8d7-45c7-b0a9-e34f9d6ba4b3)
![361DD5CB-DD6A-4D46-B251-25D4969741E7](https://github.com/user-attachments/assets/ca21faf7-a768-4a55-a89b-af7ed167cfb5)

**Reply by RyotaUshio** [reply]
@Kenotion @padanellop I cannot reproduce it on my iPad. Does it happen in a clean new vanilla vault too?

**Reply by padanellop** [reply]
Hi, I just upgraded the iPad software and now it works. Thank you, it’s an amazing plug-in.

**Reply by Sauvenay** [reply]
Hi. I try to use it with my ipad mini ios15.8.5. I have exactly the same issue.

**Reply by RyotaUshio** [reply]
@Sauvenay 

> Hi. I try to use it with my ipad mini ios15.8.5. I have exactly the same issue.

iOS15 is too old, so I'm not surprised. Can you update your iOS to the latest version, or at least to 18?

**Reply by Sauvenay** [reply]
Ah OK, should be the trouble. But I cannot update, because my ipad is 8 years old, not a recent one... Too bad. Thanks to have answer so quick!
